src/grandorgue/combinations/GOSetter.cpp

```diff
@@ -923,7 +939,7 @@ void GOSetter::Crescendo(int newpos, bool force) {
     else
       m_CrescendoExtraSets[oldIdx].clear();
     ++m_crescendopos;
-    m_crescendo[newIdx]->Push(
+    changed |= m_crescendo[newIdx]->Push(
```

Comment:
This is in need of some serious comments in my opinion. I do understand the use of bitwise OR to not short circuit the cumulative changes to changed and the need to always have the right hand part executed in this case, but it's more than likely not at all intuitive to someone reading the code to understand why we're using bitwise OR here rather than logical OR.

Reply:
I would replace it with `||=` but, unfortunally, this operator does not exist in the c++ syntax. So I replaced it with ``changed = changed || _Expression_``